package.json

```diff
@@ -134,7 +134,7 @@
     "test-suite-clean": "find test/integration/{render,query, expressions}-tests -mindepth 2 -type d -exec test -e \"{}/actual.png\" \\; -not \\( -exec test -e \"{}/style.json\" \\; \\) -print | xargs -t rm -r",
     "test-unit": "build/run-tap --reporter classic --no-coverage test/unit",
     "test-build": "build/run-tap --no-coverage test/build/**/*.test.js",
-    "test-browser": "build/run-tap --reporter spec --no-coverage test/browser/**/*.test.js",
+    "test-browser": "build/run-tap --jobs=1 --reporter spec --no-coverage -- test/browser/**/*.test.js",
```

Comment:
If I get it right, setting `--jobs=1` disable tap running tests in parallel. Does it affect the test running time?